setup.py

```diff
@@ -6,7 +6,12 @@
 setup_args.update(dict(
     name='earthsim',
     version="0.1",
+    install_requires = ['lancet'],
```

Comment:
Currently we are all just using the environment file, so that's where to add the dependency for now.

Reply:
Furthermore, putting it here would install some unrelated piece of software (https://pypi.org/project/lancet/). Unless I'm confused?

Reply:
Thanks. Fixed in 193293d.